server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/BitcoinExtension.java

```diff
@@ -107,6 +109,15 @@ public IExchange createExchange(String paramString) //(Bitstamp is in built-in e
                 String preferredFiatCurrency = FiatCurrency.USD.getCode();
                 String apiSecret = paramTokenizer.nextToken();
                 return new DVChainExchange(apiSecret, preferredFiatCurrency);
+            }else if ("lunoexchange".equalsIgnoreCase(prefix)) {
+                String apiKey = paramTokenizer.nextToken();
+                String apiSecret = paramTokenizer.nextToken();
+                String typeorder = paramTokenizer.nextToken();
+                String preferredFiatCurrency = FiatCurrency.ZAR.getCode();
```

Comment:
Not sure if calculation from ZAR to other currencies like USD will work. But if ATMs will run in ZAR it will work.